lib/modules/manager/pixi/schema.ts

```diff
@@ -1,10 +1,196 @@
+      packageName: git,
+      datasource: GitRefsDatasource.id,
+      depType: 'pypi-dependencies',
+      versioning: gitRefVersionID,
+    } satisfies PixiPackageDependency;
+  });
+
+const PypiDependencies = LooseRecord(
+  z.string(),
+  z.union([PypiDependency, PypiGitDependency]),
+)
+  .catch({})
+  .transform(collectNamedPackages);
+
+const CondaDependency = z
+  .union([
+    z.string().transform((version) => ({ version, channel: undefined })),
+    z.object({ version: z.string(), channel: z.optional(z.string()) }),
+  ])
+  .transform(({ version, channel }) => {
+    return {
+      currentValue: version,
+      versioning: condaVersion.id,
+      datasource: CondaDatasource.id,
+      depType: 'dependencies',
+      channel,
+    } satisfies PixiPackageDependency;
+  });
+
+const CondaDependencies = LooseRecord(z.string(), CondaDependency)
+  .catch({})
+  .transform(collectNamedPackages);
+
+const Targets = LooseRecord(
+  z.string(),
+  z.object({
+    dependencies: z.optional(CondaDependencies).default({}),
+    'pypi-dependencies': z.optional(PypiDependencies).default({}),
+  }),
+).transform((val) => {
+  const conda: PixiPackageDependency[] = [];
+  const pypi: PixiPackageDependency[] = [];
+  for (const value of Object.values(val)) {
+    pypi.push(...value['pypi-dependencies']);
+
+    conda.push(...value.dependencies);
+  }
+
+  return { pypi, conda };
+});
+
+const Project = z.object({
+  channels: z.array(Channel).default([]),
+});
+
+const DependenciesMixin = z
+  .object({
+    dependencies: z.optional(CondaDependencies).default({}),
+    'pypi-dependencies': z.optional(PypiDependencies).default({}),
+    target: z.optional(Targets).default({}),
+  })
+  .transform(
+    (
+      val,
+    ): { pypi: PixiPackageDependency[]; conda: PixiPackageDependency[] } => {
+      return {
+        conda: [...val.dependencies, ...val.target.conda],
+        pypi: [...val['pypi-dependencies'], ...val.target.pypi],
+      };
+    },
+  );
+
+const Features = LooseRecord(
+  z.string(),
+  z
+    .object({
+      channels: z.array(Channel).optional(),
+    })
+    .and(DependenciesMixin),
+)
+  .default({})
+  .transform(
+    (
+      features,
+    ): {
+      conda: PixiPackageDependency[];
+      pypi: PixiPackageDependency[];
+    } => {
+      const pypi: PixiPackageDependency[] = [];
+      const conda: PixiPackageDependency[] = [];
+
+      for (const feature of Object.values(features)) {
+        conda.push(
+          ...feature.conda.map((item) => {
+            return {
+              ...item,
+              channels: feature.channels,
+            };
+          }),
+        );
+
+        pypi.push(...feature.pypi);
+      }
+
+      return { pypi, conda };
+    },
+  );
 
 /**
- * config of `pixi.toml` of `tool.pixi` of `pyproject.toml`
+ * `$` of `pixi.toml` or `$.tool.pixi` of `pyproject.toml`
  */
-export const PixiConfigSchema = z.object({});
+export const PixiConfigSchema = z
+  .object({ feature: Features })
```

Comment:
```suggestion
  .object({
    feature: Features.optional(),
    workspace: Project.optional(),
    project: Project.optional(),
  })
```
Is the features block really required?

Reply:
`Features` has a `default({})`, but move it here make more sence